documentation/src/main/asciidoc/querylanguage/Preface.adoc

```diff
@@ -9,7 +9,7 @@ Hibernate 6 is a major redesign of the world's most popular and feature-rich ORM
 The redesign has touched almost every subsystem of Hibernate, including the APIs, mapping annotations, and, above all else, the query language.
 
 This is the second time Hibernate Query Language has been completely reimplemented from scratch, but the first time in more than fifteen years.
-In this new incarnation, HQL is far more powerful, and the HQL compiler much more robust.
+In this new incarnation, HQL is far more powerful, and the HQL compiler becomes much more robust.
```

Comment:
This sentence is perfectly fine as it is. I was not until this moment really aware that English lets me skip a verb when it's repeated. but it does.

Reply:
<img width="663" alt="Screenshot 2024-09-13 at 2 52 54 PM" src="https://github.com/user-attachments/assets/f6a0cc3d-0e1e-498f-9938-842e7fa46ed6">

See, even ChatGPT agrees!

Reply:
haha, a creative usage of ChatGPT in GitHub code review?
But I guess we don't need to revert it back if the change is for good?

Reply:
I like it how it was before. It's a more interesting use of the English language.

Reply:
I reverted. English language usage innotation is one of the reasons I love your articles!